data/campaigns/Sceptre_of_Fire/scenarios/7_Outriding_the_Outriders.cfg

```diff
@@ -208,6 +208,15 @@
             [/have_unit]
         [/filter_condition]
 
+        [fire_event]
+            name=reinforcements
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=reinforcements
+        first_time_only=no
+
```

Comment:
Is the event actually used more than once? If not, why not just inline the work and save the overhead?

Reply:
I split it for readability. Though, while testing I indeed triggered it twice. Currently the code shows the same dialog again, will change it to show no dialogue the 2nd time.